solr/modules/ltr/src/java/org/apache/solr/ltr/LTRScoringQuery.java

```diff
@@ -452,6 +453,12 @@ private float makeNormalizedFeaturesAndScore() {
           modelFeatureValuesNormalized[pos] = fInfo.getValue();
         } else {
           modelFeatureValuesNormalized[pos] = feature.getDefaultValue();
+          if (ltrScoringModel instanceof MultipleAdditiveTreesModel) {
```

Comment:
we should avoid this conditional instruction based on the class, can't we do in polymorphism in any way?
Maybe the model can tell you their feature default value?
Maybe we can always use Null here?

Reply:
If I put null for all the models, we have 30 tests failing.